src/routes/extras/containers/ExtrasPage.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import compose from 'lodash/fp/compose'
+import { connect } from 'react-redux'
+
+import withFetcher from 'modules/endpoint/hocs/withFetcher'
+import setSprungbrettUrl from 'modules/sprungbrett/actions/setSprungbrettUrl'
+import setLanguageChangeUrls from 'modules/language/actions/setLanguageChangeUrls'
+
+import SprungbrettPage from './SprungbrettPage'
+import TileModel from 'modules/common/models/TileModel'
+import Tiles from 'modules/common/components/Tiles'
+import ExtraModel from 'modules/endpoint/models/ExtraModel'
+import LanguageModel from 'modules/endpoint/models/LanguageModel'
+import Failure from '../../../modules/common/components/Failure'
+
+const SPRUNGBRETT_EXTRA = 'sprungbrett'
```

Comment:
Looks a bit weird to me that the ExtrasPage has to handle sprungbrett explicitly.